src/ProjectService.Business/Commands/Project/EditProjectCommand.cs

```diff
@@ -93,21 +96,23 @@ public OperationResultResponse<bool> Execute(Guid projectId, JsonPatchDocument<E
 
             GetProjectFilter filter = new GetProjectFilter { ProjectId = projectId };
 
-            DbProject dbProject = _repository.Get(filter);
+            DbProject dbProject = _projectRepository.Get(filter);
+            IEnumerable<DbProjectUser> dbUsers = _userRepository.GetProjectUsers(projectId, false);
 
             OperationResultResponse<bool> response = new();
+            Guid id = _httpContextAccessor.HttpContext.GetUserId();
 
             if (!_accessValidator.IsAdmin() &&
-                GetDepartment(dbProject.DepartmentId, response.Errors).DirectorUserId !=
-                _httpContextAccessor.HttpContext.GetUserId())
+                GetDepartment(dbProject.DepartmentId, response.Errors).DirectorUserId != id &&
+                !dbUsers.Any(user => user.UserId == id && user.Role == 0))
```

Comment:
don't use 0 here, we have enum, use UserRole here, just convert to int

Reply:
done